plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -465,9 +467,11 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
          const auto& id = block_id ? *block_id : block->calculate_id();
          auto blk_num = block->block_num();
 
-         fc_dlog(_log, "received incoming block ${n} ${id}", ("n", blk_num)("id", id));
+         auto now = fc::time_point::now();
+         if (now - block->timestamp < fc::minutes(5) || (blk_num % 1000 == 0))
```

Comment:
Should `<` be `>` so you don't log again if the last logging is less than 5 minutes ago?

Reply:
The `if` is to determine if this is a recently produced block or one from over 5 minutes ago.

Reply:
Oh I see. I first thought you wanted to log every 1000 blocks or every 5 minutes. Could you add a comment there?